### DISCUSSION TITLE
How to quickly remove all dialog windows inside MainView

### DISCUSSION THREAD

**Discussion by ultrapre** [Stellarium/stellarium#1453] [Q&A]
Any existing function ?

**Comment by gzotti** [comment]
The Stellarium User Guide, Appendix A.3, reveals a hotkey also listed in the F1 Help menu.

**Reply by ultrapre** [reply]
You meant Crtl+T ? It just hides buttoms but not dialogs windows.

**Reply by gzotti** [reply]
Oops, I was too sure it would. Then there is no such function.

**Comment by ultrapre** [comment]
Maybe I could try to add a function into StelGui to close all dialogs in class. However, how to close dialogs of plugins together is a matter.

![image](https://user-images.githubusercontent.com/32613953/103440773-76a6c600-4c83-11eb-9c9b-be3a4fa4975f.png)

**Reply by gzotti** [reply]
If every StelDialog is connected to a certain new action message (from StelApp?), it would be forced to close. Should be simple to add. However I am not certain we really want that.
